### DISCUSSION TITLE
Глупый вопрос, но возможно ли настроить раздачу через windows хот-спот с установленным на компьютере Zapret'ом?

### DISCUSSION THREAD

**Discussion by rtk-7jsq** [bol-van/zapret#946] [Q&A] [answered]
Для лучшего понимания схемы: На компьюетере под виндой стоит Zapret. С этого компьюетра раздается инет через хот-спот. Есть ли возможность настроить это дело так, чтобы подключенные к хот-споту устройства пускали трафик через пк с Zapret'ом?

**Comment by bol-van** [comment, marked as answer]
Только если через прокси. Проходной трафик не поддерживается

**Comment by galalexander** [comment]
Спасибо. Оказался в ситуации, когда роутер не настраиваемый. Простенький вариант можно глянуть тут:
[https://dzen.ru/a/Wso3ka0PItotrgE5](url)
Порт 80 оставил

Потом подключиться на ip своего пк в сети, указав нужный порт
